lib/mix/test/mix/tasks/deps_test.exs

```diff
@@ -527,7 +535,8 @@ defmodule Mix.Tasks.DepsTest do
         Mix.Tasks.Deps.Loadpaths.run([])
       end
 
-      assert_received {:mix_shell, :error, [@overriding_msg <> _]}
```

Comment:
Actually, there was an inconsistency with the @foo behavior, because this currently work, but the example currently doesn't:
```
iex(1)> defmodule A do
...(1)> @foo "foo"
...(1)> @foo <> "a" = "fooa"
...(1)> end
** (ArgumentError) invalid write attribute syntax, you probably meant to use: @foo expression
    (elixir) expanding macro: Kernel.@/1
    iex:3: A (module)
    (elixir) expanding macro: Kernel.<>/2
    iex:3: A (module)
```